### DISCUSSION TITLE
Plugin interface are not fit to scanned window of plugin side

### DISCUSSION THREAD

**Discussion by abysalvinthomas** [apohl79/audiogridder#676] [Help]
my Audio FX interfaces are showing like this in plugin side

![Screenshot 2021-07-28 at 3 57 18 PM](https://user-images.githubusercontent.com/74089164/127308446-aeab21ca-f51a-40b5-b396-aebb630cd6d5.png)

**Comment by gerardmccafferty** [comment]
> my Audio FX interfaces are showing like this in plugin side
> 
> ![Screenshot 2021-07-28 at 3 57 18 PM](https://user-images.githubusercontent.com/74089164/127308446-aeab21ca-f51a-40b5-b396-aebb630cd6d5.png)

**Reply by gerardmccafferty** [reply]
Did you manage to get a fix for this? I have the same issue - tried different screen resolutions and scaling options but without any luck. Worked perfectly fine ok 1.1.1 but when I switched to 1.2 beta 4 - I got the large space at the top of the screen
